dashboards/blazor-web-assembly/blazor-web-assembly.json

```diff
@@ -0,0 +1,1158 @@
+{
+	"name": "Blazor Web Assembly",
+	"description": null,
+
+	"pages": [
+		{
+			"name": "Overview",
+			"description": null,
+			"widgets": [
+				{
+					"title": "",
+					"layout": {
+						"column": 1,
+						"row": 1,
+						"width": 2,
+						"height": 3
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.markdown"
+					},
+					"rawConfiguration": {
+						"text": "![NuxtJS icon](https://www.zenesys.com/getattachment/Blog/Web-Development/January-2022/Blazor-Server-vs-Blazor-Web-Assembly/MicrosoftTeams-image-(15).png?width=400&height=300)\n"
```

Comment:
please update the text here from "NuxtJS icon" to "Blazor Web Assembly icon"

Reply:
Hi @pkosanam,
Changed icon name. Kindly review it